### PR TITLE
Remove aliases

### PR DESCRIPTION
**Pull request by eran-cohen**
https://monday.monday.com/boards/245345663/pulses/2983087880

Webpack aliases prevents us from importing the components directly from source in Trident, this PR fixes it (together with https://github.com/DaPulse/monday-npm-packages/pull/1657)

### CODE REVIEW

File: src/components/AvatarGroup/AvatarGroupCounterTooltipHelper.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useCallback } from "react";
-import { useKeyEvent } from "hooks";
+import { useKeyEvent } from "../../hooks";
```
Comment (orrgottlieb) [inline src/components/AvatarGroup/AvatarGroupCounterTooltipHelper.js:2]: @hadasfa lets fix this file - and make sure that we import it directly from the file and not the general expose index

File: src/components/Menu/Menu/Menu.jsx [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+import { backwardCompatibilityForProperties } from "../../../helpers/backwardCompatibilityForProperties";
 import useSubMenuIndex from "./hooks/useSubMenuIndex";
 import useOnCloseMenu from "./hooks/useOnCloseMenu";
 import useCloseMenuOnKeyEvent from "./hooks/useCloseMenuOnKeyEvent";
 import useMenuKeyboardNavigation from "./hooks/useMenuKeyboardNavigation";
 import useMouseLeave from "./hooks/useMouseLeave";
 import "./Menu.scss";
 import { useAdjacentSelectableMenuIndex } from "./hooks/useAdjacentSelectableMenuIndex";
-import { useFocusWithin } from "hooks/useFocusWithin";
+import { useFocusWithin } from "../../../hooks/useFocusWithin";
```
Comment (orrgottlieb) [inline src/components/Menu/Menu/Menu.jsx:15]: @hadasfa lets make sure that we align it to be default export

File: src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.jsx [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+import Icon from "../../../../components/Icon/Icon";
+import Check from "../../../../components/Icon/Icons/components/Check";
+import Divider from "../../../../components/Divider/Divider";
+import { NOOP } from "../../../../utils/function-utils";
+import HiddenText from "../../../../components/HiddenText/HiddenText";
+import Clickable from "../../../../components/Clickable/Clickable";
 import { MULTI_STEP_TYPES, STEP_STATUSES } from "../../MultiStepConstants";
 import { baseClassName } from "./StepIndicatorConstants";
 import "./StepIndicator.scss";
+import { useKeyEvent } from "../../../../hooks";
```
Comment (orrgottlieb) [inline src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.jsx:17]: @hadasfa directly import from file

**Review by orrgottlieb — APPROVED** [review]

File: src/components/AvatarGroup/AvatarGroupCounterTooltipHelper.js (hunk 1 above)
Comment (hadasfa) [reply src/components/AvatarGroup/AvatarGroupCounterTooltipHelper.js:2]: added to the backlog as first step. I will try to take this next iteration

File: src/components/Menu/Menu/Menu.jsx (hunk 2 above)
Comment (hadasfa) [reply src/components/Menu/Menu/Menu.jsx:15]: same

File: src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.jsx (hunk 3 above)
Comment (hadasfa) [reply src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.jsx:17]: same

**Review by hadasfa — APPROVED** [review]

**Pull request merged**